form/dynamic_form_modification.rst

```diff
@@ -490,9 +490,10 @@ sport like this::
                     $positions = null === $sport ? array() : $sport->getAvailablePositions();
 
                     $form->add('position', 'entity', array(
-                        'class'       => 'AppBundle:Position',
+                        'class' => 'AppBundle:Position',
```

Comment:
This is not related to your PR but what about dropping this doctrine notation in favor of class constants in the docs? The only advantage it has today IMO is eventually for config files with factory methods (even there I always prefer fqcn though).